new-frontiers-14/frontier-station-14
# ATM: Feedback on invalid items

## About the PR



Adds an error state to ATMs when non-speso currency (e.g. doubloons, FUCs, telecrystals) are inserted.  At time of writing, the ATM will output "ERR: INVALID" when an invalid item is inserted into the cash slot.

## Why / Balance


If somebody inserts some invalid currency, players might not think that something was inserted.

## How to test


1. Build and run local server.
2. Spawn in, open ATM.
3. ATM should display 0 in the deposit window.
4. Insert spesos.
5. ATM should display the value of the spesos in the deposit window.
6. Spawn and insert doubloons.
7. ATM should display "ERR: INVALID" in the deposit window.

## Media


![image](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/b8637515-bce3-4248-adbc-9e43dfbc54e4)


## Breaking changes


**Changelog**



- add: ATMs now display an error if non-speso items are inserted.
